docs/release_notes/9_2.md

```diff
@@ -78,8 +78,8 @@ To install Rocky Linux 9.2, head to the [download page](https://rockylinux.org/d
     * Removed packages: `binutils`, `brotli`, `dmidecode`
     * network config cleanup added
 
-* Microsoft Azure images are now published in the Shared Galleries as well as in the Marketplace. The shared galleries option provides a direct way to consume Rocky Images on Microsoft Azure without jumping through the hoops of subscribing to the image via the marketplace. In addition, this shared gallery will allow us to publish more frequently updated images to the marketplace
-* LVM cloud image variants for all types now remove `/etc/lvm/devices/system.devices` to resolve issues with PV/VG/LVs upon installation of the images due to being hardcoded to a specific device
+* Microsoft Azure images are now published in the Shared Image Gallery as well as in the Marketplace. The Shared Image Gallery option provides a direct way to consume Rocky Images on Microsoft Azure without jumping through the hoops of subscribing to the image via the Marketplace. In addition, this Shared Image Gallery will allow us to publish more frequently-updated images to the Marketplace.
```

Comment:
```suggestion
* Microsoft Azure images are now published in the Shared Image Gallery as well as in the Marketplace. The Shared Image Gallery option provides a direct way to consume Rocky Images on Microsoft Azure without jumping through the hoops of subscribing to the image via the Marketplace. In addition, this Shared Image Gallery will allow us to publish more frequently updated images to the Marketplace.
```

Reply:
we should ask @Grammaresque or @gannazhyrnova
what is best:
frequently-updated
or
frequently updated

Reply:
@SergeCroise - Serge, it depends on what exactly here needs to tell. if frequently updated - then the content will be like - we can publish this image more often, if frequently-updated - the content will be like - we can publish images that we use more often. Something like this.